WordPress/Classes/ViewRelated/Support/SupportTableViewController.swift

```diff
@@ -24,8 +25,9 @@ class SupportTableViewController: UITableViewController {
 
     // MARK: - Init
 
-    init(configuration: Configuration = .init(), style: UITableView.Style = .grouped) {
+    init(configuration: Configuration = .init(), style: UITableView.Style = .grouped, featureFlagStore: RemoteFeatureFlagStore = RemoteFeatureFlagStore()) {
```

Comment:
Got it. I'm thinking maybe then the `SupportConfiguration` is not even needed? Since I imagined the logic more or less to be:

```
switch configuration {
   case .forum: 
      // show community forum card
  case .zendesk
      // show contact support card
}
```

but the reality is different so now we have some feature flag checks inside `ViewController` and some feature flag checks in `SupportConfiguration`.

Reply:
I've left `SupportConfiguration` class and did not delete in case something comes up in the next couple of days where you might want to re-introduce